docs/developer-guide/transform-a-stream-with-ksql.rst

```diff
@@ -0,0 +1,92 @@
+.. _transform-a-stream-with-ksql:
+
+Transform a Stream With KSQL 
+############################
+
+KSQL enables *streaming transformations*, which you can use to convert
+streaming data from one format to another in real time. With a streaming

```

Comment:
Careful here, this is not true in general/by default:

> With a streaming transformation, **not only is all existing data in a stream converted**, but so is every record that subsequently arrives on the source stream.

This is only the case if you set `auto.offset.reset` to "earliest", which is not the default setting.  By default, only newly arriving data in the source stream will be transformed by a launched query.